docs/ConfigurationGuide.md

```diff
@@ -164,3 +164,46 @@ After loading the python file during the service startup each new user
 defined variable is logged for confirmation. Any failures to load
 these changes will be logged. If no new definitions are found
 indication of this will also be logged
+
+# User Customized QR Page
+
+## ConfigMap Based Overrides
+
+By default the QR page shown to users uses OpenWallet branding users
+are capable of overriding any of these files using the
+`{{ include "global.fullname" . }}-template-overrides`
+[ConfigMap](../charts/vc-authn-oidc/templates/configmap.yaml) in the
+in the helm charts. These files will override the existing files seen
+in [html-templates](../html-templates/).
+
+## Complete Directory Overrides
+
+In addition to allow for more flexibility users are welcome to build
+an image based on `ghcr.io/bcgov/vc-authn-oidc` mounting their a
+complete directory with new assets.
```

Comment:
"their a complete directory"

Reply:
The container registry address should be `ghcr.io/openwallet-foundation/acapy-vc-authn-oidc`